topics/galaxy-ui/metadata.yaml

```diff
@@ -48,6 +48,29 @@ material:
     contributors:
       - tnabtaf
 
+  -
+    title: "Jupyter: Use Jupyter notebooks in Galaxy"
+    type: "tutorial"
+    name: "galaxy-intro-jupyter"
+    zenodo_link: ""
```

Comment:
could you add the Zenodo link here as well?

Reply:
Done :)